backend/LeetWars.Notifier/LeetWars.Notifier.WebAPI/Helpers/HubManagerHelperService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using LeetWars.Notifier.Hubs.Interfaces;
+using LeetWars.Notifier.Hubs;
+using LeetWars.Notifier.WebAPI.Hubs.Interfaces;
+using LeetWars.Notifier.WebAPI.Hubs;
+using LeetWars.Notifier.WebAPI.Interfaces;
+using LeetWars.Notifier.WebAPI.Models;
+using Microsoft.AspNetCore.SignalR;
+using LeetWars.Notifier.WebAPI.Models.Helpers;
+using Newtonsoft.Json;
+using System.Text;
+
+namespace LeetWars.Notifier.WebAPI.Helpers;
+
+public class HubManagerHelperService : IHubManagerHelperService
```

Comment:
probably it would be better to have different consumers for different message types